test/net/networking_engine_unit.js

```diff
@@ -90,6 +90,8 @@ describe('NetworkingEngine', /** @suppress {accessControls} */ () => {
         backoffFactor: 0,
         fuzzFactor: 0,
         timeout: 0,
+        stallTimeout: 0,
```

Comment:
Would a new test case make sense?

Reply:
I valued it, but the problem is that there is no timeout test to be able to compare :S

Reply:
I understand.  It's tough sometimes to figure out new test cases without a similar example to work from.  Please file an issue to follow-up on this with tests, and we'll try to add some later.  Don't worry about it for now.  The code looks clear and short enough to me.  (I hope I don't regret saying that some day!  😛)

Reply:
Once merged I create the issue and link it here.